mne/cov.py

```diff
@@ -1713,6 +1713,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = dict(mag=1e15, grad=1e13, eeg=1e6)
```

Comment:
+1

2015-04-28 17:17 GMT+02:00 Alexandre Gramfort <notifications@github.com>:

> In mne/cov.py
> <https://github.com/mne-tools/mne-python/pull/2033#discussion_r29253116>:
>
> > @@ -1713,6 +1713,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
> >          thresholded to determine the rank are also returned.
> >      """
> >      picks_list = _picks_by_type(info)
> > +    if scalings is None:
> > +        scalings = dict(mag=1e15, grad=1e13, eeg=1e6)
>
> it would not make sense to import something from viz in the data container
> file. Create one such DEFAULTS variable in the file if you use it at at
> least 2 places.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2033/files#r29253116>.
>